tensorboard/plugins/pr_curve/summary.py

```diff
@@ -58,7 +57,7 @@ def op(
     predictions: A float32 `Tensor` whose values are in the range `[0, 1]`.
         Dimensions must match those of `labels`.
     num_thresholds: Number of thresholds, evenly distributed in `[0, 1]`, to
-        compute PR metrics for. Should be `>= 2`. This value should be a 
+        compute PR metrics for. Should be `>= 2`. This value should be a
```

Comment:
I liked the idea of handling these in #389. Perhaps you have a rogue commit in this PR?

Ideally, the "add backend" commit should make no changes to `summary.py`, and the "add frontend" commit (if separate) should make no changes to `pr_curves_plugin.py`/etc.